src/main/resources/builtin/materialValues.yml

```diff
@@ -4,21 +4,28 @@ default:
   lightOpacity: 255
   flameResistance: -1
   fireResistance: -1
+  baseMapColor: 44
 values:
+  # TODO: Define baseMapColor for all materials
```

Comment:
I think this would be confusing, as the material values file is incomplete for all properties. Could you add a mention for all the previously existing properties as well?